docs/commands_v2.rst

```diff
@@ -0,0 +1,310 @@
+- There are configuration options which can be set programmatically or in a fixed
+  configuration file to control behavior of the package. See the
+  `Configuration options`_ section for more details.
+
+Scenarios
+---------
+
+A scenario is an specific version of events that you like to evaluate. The
+default scenario is the `Chandra Command Events`_ sheet.
+
+Providing for alternate scenarios is a key feature of the commands archive v2.
+An example is checking for thermal propagation for assuming an ACIS CTI using
+either 3-chips or 4-chips, or no CTI at all. Such scenarios are considered
+"custom" scenarios and can be created and easily manipulated by the user.
+Examples include:
+
+- Different CTI options
+- Different "cold attitude" options
+- What if's, like what if we didn't manage to start the maneuver to 135 pitch in a NSM recovery?
+
+One special scenario is the "flight" scenario, discussed below.
+
+Flight scenario (no network)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In the case of production applications that are running on the HEAD or GRETA
+networks or where it is not possible or desirable to access the network to
+update the local loads, the special ``"flight"`` scenario can be used.
+
+The impact of selecting ``scenario="flight"`` in a commands query is that it
+disables access to network resources (Google sheets and OCCweb). This means that
+results will depend strictly on the production commands archive files
+``${SKA}/data/kadi/cmds2.h5`` and ``${SKA}/data/kadi/cmds2.pkl``. On the HEAD
+network these files are brought up to date each 10 minutes by a cron jobs, so
+using ``"flight"`` in this case is a reliable way to eliminate dependence on the
+kadi external web resources.
+
+Using the ``"flight"`` scenario is also recommended for use on GRETA
+workstations since they cannot access the Chandra Command Events Google sheet.
+
+Custom scenario example
+^^^^^^^^^^^^^^^^^^^^^^^
+
+This example shows the steps to programmatically add an ACIS CTI in the midst
+of the 2021:296 NSM recovery::
+
+
+    >>> from kadi import paths
+    >>> from kadi.commands import conf, get_cmds
+    >>> conf.commands_version = '2'
+
+    >>> cmds = get_cmds(start='2022:001')  # Ensure local cmd_events.csv is up to date
+
+    >>> path_flight = paths.CMD_EVENTS_PATH()
+    >>> path_flight
+    PosixPath('/Users/aldcroft/.kadi/cmd_events.csv')
+
+    >>> path_cti = paths.CMD_EVENTS_PATH(scenario='nsm-cti')
+    >>> path_cti
+    PosixPath('/Users/aldcroft/.kadi/nsm-cti/cmd_events.csv')
+    >>> path_cti.parent.mkdir(exist_ok=True, parents=True)
+
+    >>> from astropy.table import Table
+    >>> events_flight = Table.read(path_flight)
+    >>> events_flight.colnames
+    ['State', 'Date', 'Event', 'Params', 'Author', 'Reviewer', 'Comment']
+
+    >>> cti_event = {'State': 'definitive', 'Date': '2021:297:13:00:00',
+    ...              'Event': 'RTS', 'Params': 'RTSLOAD,1_CTI06,NUM_HOURS=12:00:00,SCS_NUM=135',
```

Comment:
I was a bit surprised that NUM_HOURS=4:00:00 wasn't compliant but not a biggie.

Reply:
I can't find a spec, but having the leading zero is probably consistent with normal usage. In any case I updated the key FOT MP process guide TWiki page like so:

<img width="821" alt="image" src="https://user-images.githubusercontent.com/348089/160171980-1c724827-b57f-43a9-82de-ddc3004f6345.png">